src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/DetachedForgotPasswordPage.java

```diff
@@ -48,10 +44,8 @@ public boolean submitButtonNotClickable() {
 
   @Override
   public void requestLinkForUsername(String username) {
-    gainFocus();
     forgotPasswordPage.requestLinkForUsername(username);
     if(forgotPasswordPage.isConfirmationDisplayed()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00108)